### ISSUE TITLE
feature request: Search option at the App Source section

### ISSUE THREAD

**Issue by maioni** [appium/appium-inspector#1004]
### Current Behavior

Why there's no SEARCH option at the App Source section (at Appium Inspector)?

It could help a lot, sometimes I can't find elements by clicking on it, so I have to copy all the source to a note text application, find something I need, and after it, goes back to inspector to make sure I can find it with a selector I created. 

An option to search here could help.

Thank you! 

### Suggested Solution

Add a Search text option to the App Source section.

After type the input, it would highlight the element or elements found. (With a find next or find previous option)

### Additional Information

I got this idea when automating a sample application SWAG LABS, after opened the menu and try to click on options (the logout option), it was selecting the elements under the menu, and not from menu. 

So I had to copy all the SOURCE, and try to find the menu elements by text, and goes back to Inspector to validate my current selector.

https://github.com/saucelabs/sample-app-mobile/releases

**Comment by jlipps** [comment]
Yeah I suppose a simple CMD+F doesn't work in the source box. Seems like a reasonable request to be able to search inside the source tree. Not sure how easy it would be to implement though! Might not be worth it if it's high effort

**Comment by sudharsan-selvaraj** [comment]
@jlipps @maioni This seems like a reasonable request and a valuable feature to have. I've managed to develop it and would love to hear your feedback. Here's the link to [PR](https://github.com/appium/appium-inspector/pull/1494) and i have added a quick demo as well.

**Comment by danieltlf** [comment]
@sudharsan-selvaraj Thank you for your PR! I really appreciate it! Good work!

**Comment by maioni** [comment]
@sudharsan-selvaraj thank you! it is really good!
